src/Bridge/Ramsey/Identifier/Normalizer/UuidNormalizer.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Ramsey\Identifier\Normalizer;
+
+use ApiPlatform\Core\Exception\InvalidIdentifierException;
+use ApiPlatform\Core\Identifier\Normalizer\IdentifierDenormalizerInterface;
+use Ramsey\Uuid\Exception\InvalidUuidStringException;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * Denormalizes an UUID string to an instance of Ramsey\Uuid.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
+ */
+final class UuidNormalizer implements IdentifierDenormalizerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function denormalize($data, $class, $format = null, array $context = [])
+    {
+        try {
+            return Uuid::fromString($data);
+        } catch (InvalidUuidStringException $e) {
+            throw new InvalidIdentifierException($e->getMessage());
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function supportsDenormalization($data, $type, $format = null)
+    {
+        return UuidInterface::class === $type || is_subclass_of($type, UuidInterface::class);
```

Comment:
Maybe can you use `is_a` to remove one test (IMO it should be similar in term of perf).

Reply:
`is_a` doesn't work with `is_a(Uuid::class, UuidInterface::class)` although Uuid does implement the class.